app/react/kubernetes/applications/application.service.ts

```diff
@@ -29,57 +31,39 @@ export async function getApplicationsForCluster(
   environmentId: EnvironmentId,
   namespaceNames?: string[]
 ) {
-  try {
-    if (!namespaceNames) {
-      return [];
-    }
-    const applications = await Promise.all(
-      namespaceNames.map((namespace) =>
-        getApplicationsForNamespace(environmentId, namespace)
-      )
-    );
-    return applications.flat();
-  } catch (e) {
-    throw parseAxiosError(
-      e as Error,
-      'Unable to retrieve applications for cluster'
-    );
+  if (!namespaceNames) {
+    return [];
   }
+  const applications = await Promise.all(
```

Comment:
removed try catch from here?

Reply:
They are still being shown from the axios services which should have 'try catch' blocks. From testing, I found that the parsed error messages got overwritten by a function like parseAxiosError or new PortainerError, and it's better to thow the parsed error as it is for it to be shown in the error messages with the correct details.

So I used the pattern:
```ts
try {
...
} catch (e) {
  throw e
}
```
To make sure the error gets to the useQuery error handling.

But this turns out to be the same as not having a try catch at all. https://eslint.org/docs/latest/rules/no-useless-catch

So I have removed try catch from functions that call axios functions that already have the correct error parsing.

Reply:
So for functions that call axios directly - try catch is needed with correct error parsing

For functions that call these functions with correct error parsing - try catch is not needed